extensions/wikia/AdEngine/src/slots.js

```diff
@@ -53,43 +56,35 @@ export function getSlotsContext(legacyContext, skin) {
 					loc: 'footer'
 				},
 				viewportConflicts: ['TOP_BOXAD']
-			}
-		},
-		mercury: {
-			'BOTTOM_LEADERBOARD': {
-				disabled: false,
-				slotName: 'BOTTOM_LEADERBOARD',
-				sizes: [
-					{
-						viewportSize: [0, 0],
-						sizes: [[320, 50], [300, 250], [300, 50]]
-					}
-				],
-				options: {},
-				defaultSizes: [[2, 2]],
-				targeting: {
-					pos: ['BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER'],
-					loc: 'footer'
-				}
 			},
-			'MOBILE_TOP_LEADERBOARD': {
+			INCONTENT_PLAYER: {
 				disabled: false,
-				slotName: 'MOBILE_TOP_LEADERBOARD',
-				sizes: [
-					{
-						viewportSize: [0, 0],
-						sizes: [[320, 480]]
-					}
-				],
-				options: {},
-				defaultSizes: [[2, 2]],
+				slotName: 'INCONTENT_PLAYER',
+				options: {
+					isVideoMegaEnabled: true
+				},
+				defaultSizes: [[1, 1]],
 				targeting: {
-					pos: 'MOBILE_TOP_LEADERBOARD',
-					loc: 'top'
-				}
-			}
-		}
-	};
+					pos: 'INCONTENT_PLAYER',
+					loc: 'middle',
+				},
+				videoAdUnit: '/{custom.dfpId}/wka1b.{adGroup}/{adProduct}{audioSegment}/{custom.device}/{targeting.skin}-{targeting.s2}/{custom.dbNameElement}-{targeting.s0}'
+			},
+		};
+	},
+
+	setupSlotVideoAdUnit(adSlot, params) {
+		const adProductInfo = getAdProductInfo(adSlot.getSlotName(), params.type, params.adProduct);
```

Comment:
It looks very similar to `src/ad-unit-builder.js` -> `build()`. Does it do sth different?